rsts/community/index.rst

```diff
@@ -118,3 +118,332 @@ Flyte maintainers are available on Zoom for you to ask anything, weekly on Wedne
     roadmap
     Frequently Asked Questions <https://github.com/flyteorg/flyte/discussions/categories/q-a>
     troubleshoot
+    
+Flyte Documentation Style Guide
+-------------------------------
+
+Overview
+^^^^^^^^
+
+This guide is an overview of style guidelines for the documentation. 
+
+.. note::
+    These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document.
+    
+Documentation formatting standards
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Usage of upper camel case
+_________________________
+- When you refer specifically to interacting with an API object, use `UpperCamelCase <https://en.wikipedia.org/wiki/Camel_case>`_, also known as Pascal case. You may see different capitalization, such as "Flytectl", in the `API Reference <https://docs.flyte.org/en/latest/reference/index.html>`_. 
+
+- When referring to any of the libraries in the Flyte ecosystem, prefer un-capitalized inline code markup format instead of capitalized plain-text format.
+
+**Example:**
+
+.. code-block:: text
+
+    “`flytekit` is an SDK for Flyte” is preferred over “Flytekit is an SDK for Flyte”.
```

Comment:
Can you indent the example blocks so that they look like sub-items of the bullet list?